chest/core.py

```diff
@@ -154,7 +154,7 @@ def __getitem__(self, key):
             if key in self.inmem:
                 value = self.inmem[key]
             else:
-                if key not in self._keys:
+                if key not in self._keys.keys():
```

Comment:
The old version is probably best `in my_dict` actually checks for membership in the keys.  Adding the `.keys()` method creates an explicit list.